packages/site/src/pages/index.js

```diff
@@ -52,12 +46,20 @@ type State = {
 }
 
 class IndexPage extends React.Component<Props, State> {
-  state = { mode: 'string', code: stringCode }
+  state = {
+    mode: 'string',
+    code: this.props.data.allMarkdownRemark.edges[0].node.hast.children[0]
+      .children[0].value
+  }
   render() {
+    const nodes = this.props.data.allMarkdownRemark.edges[0].node.hast.children
```

Comment:
`hast`? Not sure what it is just looks out of place

Reply:
https://github.com/syntax-tree/hast, It's how we can render custom components in markdown.